From dfd404c63e937d215d65a4154738e1ddd9c894fd Mon Sep 17 00:00:00 2001
From: Silvio Traversaro <silvio@traversaro.it>
Date: Wed, 23 Aug 2023 15:58:52 +0200
Subject: [PATCH] Fix bzip2 name to match the one of conda-forge on Windows

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index b2e77241..b6e1cb6a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -347,7 +347,7 @@ foreach(LIB ${LIBS_PRIVATE})
   set(LIBS "${LIBS} -l${LIB}")
 endforeach()
 STRING(CONCAT zlib_link_name "-l" ${ZLIB_LINK_LIBRARY_NAME})
-string(REGEX REPLACE "-lBZip2::BZip2" "-lbz2" LIBS ${LIBS})
+string(REGEX REPLACE "-lBZip2::BZip2" "-lbzip2" LIBS ${LIBS})
 string(REGEX REPLACE "-lLibLZMA::LibLZMA" "-llzma" LIBS ${LIBS})
 if(ENABLE_ZSTD)
   string(REGEX REPLACE "-l${zstd_TARGET}" "-lzstd" LIBS ${LIBS})
